apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -515,6 +515,7 @@ SOMAExperimentAxisQuery <- R6::R6Class(
     #' @param obsm_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mlayers()}
     #' @param varm_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mlayers(axis = 'varm')}
     #' @param obsp_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_players()}
+    #' @param drop_levels Drop unused levels from \code{obs} and \code{var} factor columns
```

Comment:
I do not use these patterns when I write docs so pardon the naive question but why do all other params here follow a pattern and this one doesn't?

Reply:
The previous ones used `\Sexpr` for templated docs that are similar but with minor variations between different methods as `@template` does not work with R6; however, the use of `\Sexpr` is the cause of the "installing package to process help pages" step when building tiledbsoma-r from source. As this step has been a source of complaint, I've resorted to copy-pasting docs and will eventually switch out all `\Sexpr` to copy-pasted docs to make building tiledbsoma-r easier

Reply:
I am 100% on board with the simplification and robustification.